### PR TITLE
[IMP] account_credit_control: Set multicompany usage. Remove warnings

### PR DESCRIPTION
**Pull request by etobella**
I have detected that the generation was not working properly on multicompany environments, this PR tries to solve the issue

### CODE REVIEW

**Review by jarroyomorales — APPROVED** [review]
Code review LGTM! :+1:

File: account_credit_control/models/credit_control_policy.py [hunk 1]
```diff
@@ -67,23 +67,24 @@ def _compute_policy_sign(self):
...
     @api.multi
-    def _move_lines_domain(self, controlling_date):
+    def _move_lines_domain(self, credit_control_run):
         """ Build the default domain for searching move lines """
         self.ensure_one()
         domain = [
             ('account_id', 'in', self.account_ids.ids),
             ('reconciled', '=', False),
             ('partner_id', '!=', False),
+            ('company_id', 'child_of', credit_control_run.company_id.id),
```
Comment (pedrobaeza) [inline account_credit_control/models/credit_control_policy.py]: Why is this needed? This is controlled by record rules, and this is not run in sudo mode.

Comment (etobella) [reply account_credit_control/models/credit_control_policy.py]: Let's assume you have Company A and its Child Company B and you are working on company A. By rules, you can create a Credit Control Run for company B, but when executed, It generates the moves for A and B. This solves this kind of issue

Comment (pedrobaeza) [reply account_credit_control/models/credit_control_policy.py]: But why `child_of`? If the credit control run is for A, it will add moves for A and B.

Comment (etobella) [reply account_credit_control/models/credit_control_policy.py]: True, I assumed it was the right option, but we can change it if you think it is better to make the company forced (both options are right for me)

Comment (pedrobaeza) [reply account_credit_control/models/credit_control_policy.py]: I would say to do it strict, and please put a comment depicting the use case you have mentioned for avoiding later doubts.

Comment (etobella) [reply account_credit_control/models/credit_control_policy.py]: Done!

**Review by pedrobaeza — APPROVED** [review]

**Comment by pedrobaeza** [comment]
/ocabot merge patch

Please FW port it to v13
